auth_tkt/helpers.py

```diff
@@ -3,11 +3,11 @@
 
 
 def get_ticket_data(ticket, authtkt_secret, crypted_cookie_secret=None,
-                    timeout=7200, encoding='utf-8'):
+                    timeout=7200, encoding='utf-8', algo='MD5'):
```

Comment:
I think we should probably rename `algo` to `digest_type` for similarity to mod_auth_tkt's `TKTDigestType` parameter.